recipes/blue/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "blue" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/blue-{{ version }}.tar.gz
+  sha256: da06d09f8417ad475e21a26dd0a7dae6771146fdd7f5760450183183230f9ef1
+
+build:
+  entry_points:
+    - blue=blue:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - tox
```

Comment:
```suggestion
```
tox is only required for tests?

Reply:
It is used in the setup.py but let's see if it gets imported or not.